tests/buildkite/schedule_and_publish/test_run_benchalerts.py

```diff
@@ -0,0 +1,119 @@
+import json
```

Comment:
These tests were essentially copied over from the deleted test files (with slight modification)